### PR TITLE
Fix for node.js v20 [ui]

### PR DESCRIPTION
**Pull request by slavas490**
https://github.com/ProxeusApp/proxeus-core/issues/366

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,6 +71,7 @@ dependencies=go curl
 mocks=main/handlers/blockchain/mock/adapter_mock.go storage/mock/interfaces.go
 bindata=main/handlers/assets/bindata.go test/assets/bindata.go
 golocalimport=github.com/ProxeusApp/proxeus-core
+uiextraflags=NODE_OPTIONS=--openssl-legacy-provider
```
Comment (loleg) [inline Makefile:74]: It seems to me that it needs to be a MacOS, Node 17 only fix.

**Review by loleg — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (slavas490) [reply Makefile:74]: [circleci](https://github.com/ProxeusApp/proxeus-core/blob/main/.circleci/config.yml) still running it with 16 node.js

maybe we should update our circleci config to be able to use this flag

Comment (loleg) [reply Makefile:74]: Let's upgrade CircleCI to 20.

Comment (loleg) [reply Makefile:74]: As discussed today, we are probably not quite ready for Node 20.x. I've updated this in #397

**Comment by loleg** [comment]
Let's close #397 and get back to this after, OK?

**Pull request closed without merging**